app/scripts/modules/ecs/src/serverGroup/configure/wizard/serviceDiscovery/ServiceDiscovery.tsx

```diff
@@ -39,13 +39,12 @@ export class ServiceDiscovery extends React.Component<IServiceDiscoveryProps, IS
       serviceDiscoveryAssociations: cmd.serviceDiscoveryAssociations,
       serviceDiscoveryRegistriesAvailable:
         cmd.backingData && cmd.backingData.filtered ? cmd.backingData.filtered.serviceDiscoveryRegistries : [],
-      useTaskDefinitionArtifact: cmd.useTaskDefinitionArtifact ? true : false,
+      useTaskDefinitionArtifact: cmd.useTaskDefinitionArtifact,
     };
-
```

Comment:
nit: can we add the separator back here as well (improves readability)

Reply:
change has been made.